### PR TITLE
(feature) Adds optional keep_alived firewall rules to task 3.5.1.6

### PR DESCRIPTION
**Pull request by jvleminc**
- (feature) Adds optional keep_alived firewall rules to task 3.5.1.6
- (chore) Changes some "false" to "no", for consistency reasons. Adds some comments. Changes variable name to use "firewall" and not "ufw".

### CODE REVIEW

File: tasks/section_3_Network_Configuration.yaml [hunk 1]
```diff
@@ -549,27 +549,39 @@
...
         rule: "{{ item.rule }}"
         port: "{{ item.port }}"
         proto: "{{ item.proto }}"
-      with_items: "{{ list_of_ports_to_allow }}"
+      with_items: "{{ firewall_list_of_ports_to_allow }}"
+    - name: 3.5.1.6 Ensure firewall rules exist for all open ports | keep_alived (1)
+      ufw:
+        # note: Ansible doesn't support "proto: vrrp"
+        rule: allow
+        from_ip: 224.0.0.18
```
Comment (alivx) [inline tasks/section_3_Network_Configuration.yaml:577]: Can we set a dynamic config for this static IP

File: tasks/section_3_Network_Configuration.yaml [hunk 2]
```diff
@@ -549,27 +549,39 @@
...
+      when: firewall_allow_keep_alive
+    - name: 3.5.1.6 Ensure firewall rules exist for all open ports | keep_alived (2)
+      ufw:
+        # note: Ansible doesn't support "proto: vrrp"
+        rule: allow
+        to_ip: 224.0.0.18
```
Comment (alivx) [inline tasks/section_3_Network_Configuration.yaml:583]: Can we set a dynamic config for this static IP

**Review by alivx — CHANGES_REQUESTED** [review]

File: tasks/section_3_Network_Configuration.yaml (hunk 1 above)
Comment (jvleminc) [reply tasks/section_3_Network_Configuration.yaml:577]: No, this is the broadcast IP used by VRRP.

https://tools.ietf.org/html/rfc3768:
```
The IP multicast address as assigned by the IANA for VRRP is:

      224.0.0.18

   This is a link local scope multicast address.  Routers MUST NOT
   forward a datagram with this destination address regardless of its
   TTL.
```

File: tasks/section_3_Network_Configuration.yaml (hunk 2 above)
Comment (jvleminc) [reply tasks/section_3_Network_Configuration.yaml:583]: Idem

**Pull request merged**